source/set-up/shared-channels.rst

```diff
@@ -0,0 +1,193 @@
+Shared Channels (Experimental) (E20)
+====================================
+
+Shared Channels bring people together from multiple Mattermost, such as teams collaborating with external partners, and customers using multiple Mattermost instances in a federated architecture. 
+
+Mattermost System Admins use slash commands to establish secure connections between Mattermost instances, then invite secured connections to shared channels.
+
+Shared channels behave like regular channels, offering the same user experience and functionality as Public and Private channels. All members of secure connections, including local members and remote members, can `send and receive messages <https://docs.mattermost.com/help/messaging/sending-messages.html#sending-and-receiving-messages>`__, `use emojis <https://docs.mattermost.com/help/messaging/emoji.html>`__ to react to messages, `share files <https://docs.mattermost.com/help/messaging/attaching-files.html>`__, and `search message history <https://docs.mattermost.com/help/getting-started/searching.html#searching-for-message-contents>`__. Content is synchronized in real-time across all participating Mattermost instances. 
```

Comment:
Great point! In this instance, the term "secure connection" is functioning as an informal noun, rather than as a product name (formal noun) that would otherwise be capitalized.